js/collections/Results.js

```diff
@@ -0,0 +1,54 @@
+import { integerToDecimal } from '../utils/currency';
```

Comment:
> Seems like some spacing is needed between the icon and text:

Glad you bought it up. I feel the same way. It looks like the spaces are in the DOM, but they're not outputting for some reason. 

> Should we be allowing search within this drop down?

Yeah, not super useful. In most cases, the dropdown would probably be under 5 options.

> The left border of the search button is not changing on hover:

If it's easier to put a gap between the input and button, that should be fine too.